plugins/grpc-ecosystem/gateway/v2.15.2/separate_pkg_additional_imports.patch

```diff
@@ -3,7 +3,7 @@ index 9cd14cc7..fe6c7199 100644
 --- a/internal/descriptor/registry.go
 +++ b/internal/descriptor/registry.go
 @@ -146,6 +146,13 @@ type Registry struct {
-
+ 
```

Comment:
I'm guessing my version of `git diff` is generating slightly different changes than before.

Reply:
reviewing a diff of a diff is fun.

Reply:
Yeah the best way might be to check out v2.16.0 of grpc-gateway and apply it there so you can review the changes there. That's what I did originally.